docs/dev-guide/contributing/bugs.rst

```diff
@@ -49,6 +48,22 @@ necessary, the bug may be commented on requesting more information or
 clarification from the reporter. When a bug has enough information, its
 *Priority* rating set and is marked as triaged using the *Triaged* boolean.
 
+Security Disclosures
+--------------------
+
+We take security issues seriously and welcome responsible disclosure of security vulnerabilities
+in Pulp. Please email `pulp-security@redhat.com` (a private address for the Pulp Security Team)
+with all reports.
+
+Your report should include:
+
+* Pulp version
+* A vulnerability description
+* Reproduction steps
+
+A member of the Pulp Security Team will confirm the vulnerability, determine its impact, and
+develop a fix.
```

Comment:
Perhaps a "feel free to submit a patch with your disclosure" or similar would be good. Security researchers often like to include patches.